bin/do_xdmat.py

```diff
@@ -64,8 +64,8 @@ def calc_dmat(p):
     parser.add_argument('--z-ref', type = float, default = 2.25, required=False,
                     help = 'reference redshift')
 
-    parser.add_argument('--rej', type = float, default = 1., required=False,
-                    help = 'reference redshift')
+    parser.add_argument('--rej', type = float, default = 0., required=False,
+                    help = 'fraction rejected: 0=no rejection, 1=all rejection')
 
```

Comment:
this change would make the --rej option inconsistent with what's in do_dmat.... why do you want to do this change?